style.css

```diff
@@ -493,6 +515,101 @@ button {
   color: var(--neutral5);
 }
 
+/******************************  Pop Up  Ovelay  Starts  ****************************/
+.popup-container {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.3);
+}
```

Comment:
- [ ] Great implementation of the popup window :clap:  However You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the [templates in Figma.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your Design: 

![bad](https://user-images.githubusercontent.com/57562869/151190022-b32a11d7-c05a-47b9-bb65-5e2a4261075b.png)

Figma Design:

![coll](https://user-images.githubusercontent.com/57562869/151190080-8b313bdb-f077-4508-a010-e4945d66cac8.png)